docs/examples/python_sdk_examples.ipynb

```diff
@@ -0,0 +1,355 @@
+      "Pending\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(deployed_isvc[\"status\"][\"modelStatus\"][\"states\"][\"activeModelState\"])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check `InferenceService` status once its ready\n",
+    "It should now be in a `Loaded` state."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kserve.wait_isvc_ready(name, namespace=namespace)\n",
+    "deployed_isvc = kserve.get(name, namespace=namespace)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loaded\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(deployed_isvc[\"status\"][\"modelStatus\"][\"states\"][\"activeModelState\"])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Update `InferenceService` and point it to a different model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "updated_spec=V1beta1InferenceServiceSpec(\n",
+    "    predictor=V1beta1PredictorSpec(\n",
+    "        tensorflow=V1beta1TFServingSpec(\n",
+    "            protocol_version=protocol_version,\n",
+    "            storage=V1beta1StorageSpec(\n",
+    "                key='localMinIO',\n",
+    "                path='tensorflow/mnist.savedmodel'\n",
+    "                )\n",
+    "            )\n",
+    "        )\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "updated_isvc = V1beta1InferenceService(api_version= constants.KSERVE_V1BETA1,\n",
+    "                          kind=constants.KSERVE_KIND,\n",
+    "                          metadata=client.V1ObjectMeta(name=name, namespace=namespace),\n",
+    "                          spec=updated_spec)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "replace_isvc = kserve.replace(name, updated_isvc, namespace=namespace)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "key:  localMinIO\n",
+      "path:  tensorflow/mnist.savedmodel\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"key: \", replace_isvc[\"spec\"][\"predictor\"][\"tensorflow\"][\"storage\"][\"key\"])\n",
+    "print(\"path: \", replace_isvc[\"spec\"][\"predictor\"][\"tensorflow\"][\"storage\"][\"path\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kserve.wait_isvc_ready(name, namespace=namespace)\n",
```

Comment:
Maybe we could add another text cell above this one to explain that/why we need to wait for isvc to be ready. Even though the method name is indicating it.